rootfs/api/models/app.py

```diff
@@ -191,6 +191,7 @@ def create(self, *args, **kwargs):  # noqa
 
         # create required minimum resources in k8s for the application
         namespace = self.id
+        ingress = self.id
```

Comment:
Doesn't have to happen in this PR, but a neat python trick is to set these all to self.id in one shot:

```
namespace = ingress = service = self.id
```

As shown here:

```
><> python3
Python 3.5.2 (default, Aug  8 2016, 12:14:07) 
[GCC 4.9.2] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a = b = c = 3
>>> a
3
>>> b
3
>>> c
3
>>>
```